### PR TITLE
Remove user escrow

### PR DESCRIPTION
**Pull request by hieronx**

### CODE REVIEW

File: test/fork/Fork.t.sol [hunk 1]
```diff
@@ -105,9 +104,9 @@ contract ForkTest is Test {
                 assertEq(Escrow(escrow).wards(admin), 0);
 
                 // UserEscrow
-                assertEq(UserEscrow(userEscrow).wards(root), 1);
-                assertEq(UserEscrow(userEscrow).wards(deployer), 0);
-                assertEq(UserEscrow(userEscrow).wards(admin), 0);
+                // assertEq(UserEscrow(userEscrow).wards(root), 1);
+                // assertEq(UserEscrow(userEscrow).wards(deployer), 0);
+                // assertEq(UserEscrow(userEscrow).wards(admin), 0);
```
Comment (AStox) [inline test/fork/Fork.t.sol:109]:
```suggestion
```

File: test/fork/Fork.t.sol [hunk 2]
```diff
@@ -65,12 +64,12 @@ contract ForkTest is Test {
...
                 // InvestmentManager
                 assertEq(address(InvestmentManager(investmentManager).escrow()), escrow);
-                assertEq(address(InvestmentManager(investmentManager).userEscrow()), userEscrow);
+                // assertEq(address(InvestmentManager(investmentManager).userEscrow()), userEscrow);
                 assertEq(address(InvestmentManager(investmentManager).gateway()), gateway);
                 assertEq(address(InvestmentManager(investmentManager).poolManager()), poolManager);
                 assertEq(InvestmentManager(investmentManager).wards(poolManager), 1);
                 assertEq(Escrow(escrow).wards(investmentManager), 1);
-                assertEq(UserEscrow(userEscrow).wards(investmentManager), 1);
+                // assertEq(UserEscrow(userEscrow).wards(investmentManager), 1);
```
Comment (AStox) [inline test/fork/Fork.t.sol:72]:
```suggestion
```

File: test/fork/Fork.t.sol [hunk 3]
```diff
@@ -65,12 +64,12 @@ contract ForkTest is Test {
 
                 // InvestmentManager
                 assertEq(address(InvestmentManager(investmentManager).escrow()), escrow);
-                assertEq(address(InvestmentManager(investmentManager).userEscrow()), userEscrow);
+                // assertEq(address(InvestmentManager(investmentManager).userEscrow()), userEscrow);
```
Comment (AStox) [inline test/fork/Fork.t.sol:67]:
```suggestion
```

**Review by AStox — APPROVED** [review]
So much nicer! 👌

File: test/fork/Fork.t.sol (hunk 1 above)
Comment (hieronx) [reply test/fork/Fork.t.sol:109]: I'll remove it though

**Review by ilinzweilin — APPROVED** [review]

**Pull request merged**